internal/driver/onvifdiscovery.go

```diff
@@ -133,6 +132,13 @@ func (d *Driver) createDiscoveredDevice(onvifDevice onvif.Device) (sdkModel.Disc
 		device.Protocols[OnvifProtocol][DeviceStatus] = UpWithAuth
 		device.Protocols[OnvifProtocol][LastSeen] = time.Now().Format(time.UnixDate)
 
+		netInfo, err := d.getNetworkInterfaces(device)
+		if err != nil {
+			d.lc.Warnf("failed to get the network information for the camera %s, %v", endpointRefAddr, edgexErr)
```

Comment:
This should be moved below the deviceName name assignment, and the deviceName used in the error message instead of endpointRefAddr